### ISSUE TITLE
uncaughtException is not emitted after using async-hook

### ISSUE THREAD

**Issue by RobinQu** [AndreasMadsen/async-hook#10]
```
asyncHook.enable();
process.nextTick(function () {
  throw new Error('boom');
});
```

No error is thrown and process exits with status 0.

**Comment by RobinQu** [comment]
Connect with https://github.com/AndreasMadsen/async-hook/pull/11

**Comment by AndreasMadsen** [comment]
Fixed by #11 and landed in 129af3d9f4961ecb1599c1232988fe87aece61ee
